docs/standard/serialization/binaryformatter-migration-guide/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "BinaryFormatter migration guide"
+description: "This guide covers the deprecation and removal of BinaryFormatter from .NET and recommends migration paths."
+ms.date: 5/31/2024
+no-loc: [BinaryFormatter, Serialization]
+helpviewer_keywords:
+  - "BinaryFormatter"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+---
+
+# BinaryFormatter migration guide
+
+> [!CAUTION]
+> We strongly recommend against using BinaryFormatter due to the [associated security risks](#whats-the-risk-in-using-binaryformatter). Existing users [should migrate away from BinaryFormatter](#migration-topics).
+
+Starting with .NET 9, we no longer ship an implementation of BinaryFormatter. The APIs are still present, but their implementation always throws an exception, regardless of project type. Hence, setting the existing backwards compatibility flag is no longer sufficient to use BinaryFormatter.
```

Comment:
I tried to describe what the in-box situation is. This now reads a bit strange because in conjunction with the next sentence it says the "the unsupported implementation always throws". The implementation we ship in-box is argubaly supported (since it always throws) -- what's unsupported is the OOB implementation which still functions :-)


```suggestion
Starting with .NET 9, we no longer include an implementation of BinaryFormatter in the runtime. The APIs are still present, but their implementation always throws an exception, regardless of project type. Hence, setting the existing backwards compatibility flag is no longer sufficient to use BinaryFormatter.
```